.gitignore

```diff
@@ -3,4 +3,6 @@ ld_extension.code-workspace
 package.json
 .eslintrc.json
 package-lock.json
-node_modules/
\ No newline at end of file
+node_modules/
```

Comment:
You should ignore node_modules

But  keep in package, eslintrc, prettierrc and prettierignore - otherwise another develop cannot follow the same standards as you